### PR TITLE
[release-0.2] docs: update charts installation instructions

### PR DESCRIPTION
**Pull request by fmuyassarov**

### CODE REVIEW

File: docs/resource-policy/installation.md [hunk 1]
```diff
@@ -41,38 +41,44 @@ following components: DaemonSet, ConfigMap, CustomResourceDefinition, and RBAC-r
...
+1. Install the plugin. Replace release version with the desired version. If you wish to
+   provide custom values to the Helm chart, refer to the [table](#helm-parameters) below,
+   which describes the available parameters that can be modified before installation.
+   It's important to note that specifying the namespace (using `--namespace` or `-n`) is
+   crucial when installing the Helm chart. If no namespace is specified, the manifests
+   will be installed in the default namespace.
 
     ```sh
-    helm install topology-aware --namespace kube-system deployment/helm/topology-aware/
+    helm install nri-plugins/nri-resource-policy-topology-aware --namespace kube-system --version v0.2.0
```
Comment (marquiz) [inline docs/resource-policy/installation.md:72]: Missing name for the deployment.

Also, I think that we'd better not hard code the version in the md file(s) but specify the version in `docs/conf.py` (in `myst_substitutions`) and use variable substitution here

```suggestion
    helm install topology-aware nri-plugins/nri-resource-policy-topology-aware --namespace kube-system --version {{ version }}
```

**Review by marquiz — COMMENTED** [review]
Thanks @fmuyassarov for the update. One comment but otherwise looks good

**Comment by marquiz** [comment]
I think we can just backport #147 here and address the version thingie later if needed

File: docs/resource-policy/installation.md (hunk 1 above)
Comment (fmuyassarov) [reply docs/resource-policy/installation.md:72]: I'm not sure if how docs/conf.py is used. Please have a look @marquiz . I'm fine even to remove it if needed.

Comment (fmuyassarov) [reply docs/resource-policy/installation.md]: I excluded the version for now and we can come back to it later on.

**Review by marquiz — APPROVED** [review]
LGTM

**Pull request merged**